vm/src/stdlib/ssl.rs

```diff
@@ -1257,3 +1260,83 @@ impl Write for SocketStream {
         socket.flush()
     }
 }
+
+#[cfg(target_os = "android")]
+mod droid {
```

Comment:
is there any reason this is not `android` but `droid`? are there more droids?

Reply:
sorry about my bad naming style, i'll fix this soon.

Reply:
don't worry. I was actually wondered how it is named because I don't know about them well.